modules/ROOT/pages/connected-apps-overview.adoc

```diff
@@ -0,0 +1,36 @@
+= Connected Apps
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can connect apps to Anypoint Platform without granting them unrestricted access to data or having to share sensitive credentials.
+When you grant an app access, you have direct control over its data access, usage, and credentials.
+You can audit the use of data by connected apps, and you can also revoke access at any time.
+
+== Authentication
+
+Use OAuth 2.0 and OpenID to authorize apps to access resources using tokens or Single Sign-On (SSO).
+Additionally, you can share account data with third-party applications.
+For more information on which users can grant access to applications using Anypoint Platform, see Connected Apps Permissions by User.
+
+== First-party and Third-party Applications
+
+To determine whether the app you want to connect is a first- or third-party application, consider the below:
+
+* First-party: 
+
+  An app that is controlled by organization to which you belong.
+
+* Third-party application: 
+
+  An app that is controlled by another organization.
+
+Applications can also be public or private.
```

Comment:
Related to the above comment, categorizing first/third party next to public/private is weird because public/private is absolute, but parties are relative; what's a first-party app to me is a third-party app to you. That is, parties aren't a property of the app alone, while visibility is.